graphhopper.sh

```diff
@@ -2,68 +2,105 @@
 (set -o igncr) 2>/dev/null && set -o igncr; # this comment is required for handling Windows cr/lf 
 # See StackOverflow answer http://stackoverflow.com/a/14607651
 
-GH_CLASS=com.graphhopper.tools.Import
 GH_HOME=$(dirname "$0")
 JAVA=$JAVA_HOME/bin/java
 if [ "$JAVA_HOME" = "" ]; then
  JAVA=java
 fi
 
-vers=$($JAVA -version 2>&1 | grep "java version" | awk '{print $3}' | tr -d \")
+vers=$($JAVA -version 2>&1 | grep "version" | awk '{print $3}' | tr -d \")
 bit64=$($JAVA -version 2>&1 | grep "64-Bit")
 if [ "$bit64" != "" ]; then
   vers="$vers (64bit)"
 fi
 echo "## using java $vers from $JAVA_HOME"
 
-CONFIG=config.properties
-if [ ! -f "config.properties" ]; then
-  cp config-example.properties $CONFIG
-fi
+function printBashUsage {
+  echo "Usage:"
+  echo "-h | --help:   display this message"
+  echo "-a | --action: must be one the following actions:"
+  echo "     --action import      creates the graph cache only, used for later faster starts"
+  echo "     --action web         starts a local server for user access at localhost:8989 and API access at localhost:8989/route"
+  echo "     --action build       creates the graphhopper web JAR"
+  echo "     --action clean       removes all JARs, necessary if you need to use the latest source (e.g. after switching the branch etc)"
+  echo "     --action measurement does performance analysis of the current source version via random routes (Measurement class)"
+  echo "     --action torture     can be used to test real world routes via feeding graphhopper logs into a GraphHopper system (Torture class)"
+  echo "-i | --input:       path to the input map file or name of the file to download"
+  echo "-ig| --gtfs:        path to the input file for public transit routing (GTFS format)"
+  echo "-o | --graph-cache: directory for graph cache output"
+  echo "-p | --profiles:    specify a comma separated list of vehicle profiles"
+  echo "-c | --config:      specify the application configuration"
+  echo "-host:              specify to which host the service should be bound"
+  echo "-port:              start web server at specific port"
+  echo "-jar:               specify the jar file (useful if you want to reuse this script for custom builds)"
```

Comment:
or should we better do something like `-h | --host` and `-p | --port` (clashes with `--profiles`) and `-j | --jar`?

Reply:
Yes, it will be nice. I think you should use -p for port, and perhaps just --profile without a short version.

Reply:
ok. Maybe `-v` for `--vehicle-profiles` but yes, `--profiles` is also ok

Reply:
-h for --host would clash with -h for help :)

Reply:
And -v should be used for version. So I currently prefer to use --host and --port and --jar